Assets/Scripts/Augment/BulletModifiers/DecalModifier.cs

```diff
@@ -57,13 +57,13 @@ public void Detach(ProjectileController projectile)
 
     private void OnHit(RaycastHit target, ref ProjectileState state)
     {
-        // Soda cans are tiny and flail around, making them unsuitable for placing bullet holes on
-        // TODO Add some way of determining which *other* objects to avoid placing decals on
-        if (target.collider.TryGetComponent<SodaCan>(out var _))
-            return;
-        // Also avoid placing decals on players, as that leads to the heebie-jeebies
+        // Avoid placing decals on players or their guns, as that leads to the heebie-jeebies
         if ((target.collider.gameObject.layer & allGunsAndPlayersMask) > 0)
             return;
+        // Avoid decals on other objects that are marked as unsuitable for placing bullet holes on
+        if (target.collider.gameObject.CompareTag("NoDecal"))
+            return;
+        // And player hitboxes
         if (target.collider.TryGetComponent<HitboxController>(out var hitbox))
             if (hitbox.health && hitbox.health.TryGetComponent<PlayerManager>(out var _))
                 return;
```

Comment:
Given we now check for the hitbox layer, maybe this line of code is superfluous?

Reply:
Not necessarily. Decals should appear on the instrument bots, and they have hitboxes w/o having players tied to them... however, player hitboxes will already be excluded by the layer check so... yeah, we _should_ be able to get rid of this check? 🤔